### DISCUSSION TITLE
Export As HTML

### DISCUSSION THREAD

**Discussion by VpkPrasanna** [mindee/doctr#919] [Ideas]
is there any way from Doctr to render or generate **HTML page from the output ?**
Input - > PDF or Image 
Output -> Rendered or Re-constructed HTML Page

**Comment by felixdittrich92** [comment]
Hi @VpkPrasanna  👋 ,
we have currently not the option to export as html. But you can try to use the tutorial to export as pdfa and than to html (note: this supports currently only straight boxes)  :)

**Reply by VpkPrasanna** [reply]
can you share some insights on how to export it as PDFA ?

**Comment by felixdittrich92** [comment]
Sure :) https://github.com/mindee/notebooks/blob/main/doctr/export_as_pdfa.ipynb
But take note there is an open PR which is currently not merged and fixes some minor stuff in the notebook i have linked :)

**Reply by VpkPrasanna** [reply]
Thanks for the notebook. It works fine but when we use the Generated PDF to convert into HTML format it is not working . is there any work around to export the doctr output into HTML page?

**Reply by felixdittrich92** [reply]
Hi @VpkPrasanna 
you can try to work with the XML output from doctr which contains all information in hOCR format.

```
xml_output = result.export_as_xml()
for output in xml_output:
  xml_bytes_string = output[0]
```

https://mindee.github.io/doctr/using_doctr/using_models.html
scroll down there is an example :)

**Reply by VpkPrasanna** [reply]
Yes , i already tried this but the result of XML is no the designated format and when we render it as HTML it is more clumsy .
